app/src/main/java/bubbleheads/buy_cook/Recipe.java

```diff
@@ -1,19 +1,18 @@
 package bubbleheads.buy_cook;
 
-import java.util.ArrayList;
-import java.util.List;
-
 public class Recipe {
     private final String recipeName;
     private final String recipeDetail;
     private final int recipePhoto;
+    private final String recipeHowToCook;
     private final Category category;
     private boolean favorite;
 
     public Recipe(final String name, final String detail,
-                  final int photo, final Category category) {
+                  final int photo, final Category category, final String howToCook) {
```

Comment:
parameter and fields use different names. It is common practise to have same name. If field is called recipeName - parameter should also be called the same